src/ui-kit/Search/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { setSearchWord } from 'redux/actions'
+
+import { func } from 'prop-types'
+import { Input } from 'antd'
+
+import { SEARCH_DEFAULT_TEXT } from '../constants'
+import './search.scss'
+
+const { Search } = Input
+
+const SearchKit = ({setSearchWord}) => {
+
+  const setCurrentYears = (value) => {
+    setSearchWord(value)
+  }
+
+  return (
+    <div>
+      <Search
+        className="ui-kit__search"
+        placeholder={SEARCH_DEFAULT_TEXT}
+        onSearch={value => setCurrentYears(value)}
```

Comment:
remember that if you have a function that receives one value, this:
 - **onSearch={value => setCurrentYears(value)}**
will always be equal to this:
- **onSearch={setCurrentYears}**

The second form is better and more readable, because you dont create a new arrow function every time